app/views/quotes/index.html.erb

```diff
@@ -1,3 +1,6 @@
 <% @quotes.each do |quote| %>
-  <%= render quote %>
+  <div class="well">
+    <%= link_to "->", quote %>
+    <%= render quote, vote: quote.votes.find_by(user: current_user) %>
```

Comment:
Does the vote need to passed in, or can it just be found off of the quote in the partial